pingdom/resource_pingdom_check_test.go

```diff
@@ -46,6 +46,7 @@ func TestAccResourcePingdomCheck_http(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "probefilters.#", "0"),
 					resource.TestCheckResourceAttr(resourceName, "userids.#", "0"),
 					resource.TestCheckResourceAttr(resourceName, "teamids.#", "0"),
+					resource.TestCheckResourceAttr(resourceName, "ipv6", "true"),
```

Comment:
if default is false how is it true in check?

Reply:
should be false. fixed